### PR TITLE
Module fixes (compiling AppInsightsModule, fixing ``main`` reference)

### PR DESCRIPTION
**Pull request by AlexBulankou**
* Adding separate project for AppInsightsModule (as dicussed offline with @kamilszostak)
* Adding separate interface-only TS project and referencing it from JavaScriptSDK and JavaScriptSDK.Module projects
* Fixing ``main`` reference in package.json
* Updating dist folder and bumping up version
This should address the blocking issues mentioned in #221

### CODE REVIEW

**Comment by MaxShehovtsov** [comment]
LGTM, but will give it another look once you merge and it's available for trying out locally.

File: Global.props [hunk 1]
```diff
@@ -90,7 +90,7 @@
     -->
     <SemanticVersionMajor>0</SemanticVersionMajor>
     <SemanticVersionMinor>22</SemanticVersionMinor>
-    <SemanticVersionPatch>17</SemanticVersionPatch>
+    <SemanticVersionPatch>18</SemanticVersionPatch>
     <!-- 
       Date when Semantic Version was changed. 
       Update for every public release.
```
Comment (kamilszostak) [inline Global.props]: Please also update the version data on line 98.

File: dist/ai.js [hunk 2]
```diff
@@ -1,3 +1,8 @@
+var __extends = (this && this.__extends) || function (d, b) {
```
Comment (kamilszostak) [inline dist/ai.js]: Did you take this file from build drop dir? I made this mistake before and Max suggested to always release from the official build drop directory (\\vscsstor\CIDrops\AIC\DC_JavaScript_master).

**Comment by kamilszostak** [comment]
:shipit:

File: dist/ai.js (hunk 2 above)
Comment (AlexBulankou) [inline dist/ai.js]: you're right, I probably did. We should use your script to deploy.

**Pull request merged**